resources/assets/js/app/models/CommentModel.js

```diff
@@ -4,7 +4,7 @@ var _ = require('underscore');
 module.exports = BaseModel.extend({
     urlRoot: '/comments',
     initialize: function (data, options) {
-        if (options.parentUrl) {
+        if (options && options.parentUrl) {
             this.parentUrl = options.parentUrl;
```

Comment:
implement this logic in BaseCollection, we have a lot of same initializers accross app